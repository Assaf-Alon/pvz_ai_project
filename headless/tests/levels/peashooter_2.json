{
    "10": [["normal", 0]],
    "20": [["normal", 0]],
    "30": [["normal", 0]],
    "40": [["normal", 0]],
    "50": [["normal", 0]],
    "60": [["normal", 0]]
}
