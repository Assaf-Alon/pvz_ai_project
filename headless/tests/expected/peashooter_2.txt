[100] Zombie being spawned at lane 0
[101] Sun autospawned. Total: 75.
[147] Zombie in (0, 9) moved to (0, 8).
[194] Zombie in (0, 8) moved to (0, 7).
[200] Zombie being spawned at lane 0
[202] Sun autospawned. Total: 100.
[203] Planted Peashooter in (0, 0). Total: 0.
[218] Pea generated by Peashooter in (0, 0).
[218] Pea in (0, 0) moved to (0, 1).
[220] Pea in (0, 1) moved to (0, 2).
[222] Pea in (0, 2) moved to (0, 3).
[224] Pea in (0, 3) moved to (0, 4).
[226] Pea in (0, 4) moved to (0, 5).
[228] Pea in (0, 5) moved to (0, 6).
[230] Pea in (0, 6) moved to (0, 7).
[230] Pea in (0, 7) attacked.
[230] Zombie in (0, 7) was damaged. HP: 161.
[230] Pea in (0, 7) removed.
[233] Pea generated by Peashooter in (0, 0).
[233] Pea in (0, 0) moved to (0, 1).
[235] Pea in (0, 1) moved to (0, 2).
[237] Pea in (0, 2) moved to (0, 3).
[239] Pea in (0, 3) moved to (0, 4).
[241] Pea in (0, 4) moved to (0, 5).
[241] Zombie in (0, 7) moved to (0, 6).
[243] Pea in (0, 5) moved to (0, 6).
[243] Pea in (0, 6) attacked.
[243] Zombie in (0, 6) was damaged. HP: 141.
[243] Pea in (0, 6) removed.
[247] Zombie in (0, 9) moved to (0, 8).
[248] Pea generated by Peashooter in (0, 0).
[248] Pea in (0, 0) moved to (0, 1).
[250] Pea in (0, 1) moved to (0, 2).
[252] Pea in (0, 2) moved to (0, 3).
[254] Pea in (0, 3) moved to (0, 4).
[256] Pea in (0, 4) moved to (0, 5).
[258] Pea in (0, 5) moved to (0, 6).
[258] Pea in (0, 6) attacked.
[258] Zombie in (0, 6) was damaged. HP: 121.
[258] Pea in (0, 6) removed.
[263] Pea generated by Peashooter in (0, 0).
[263] Pea in (0, 0) moved to (0, 1).
[265] Pea in (0, 1) moved to (0, 2).
[267] Pea in (0, 2) moved to (0, 3).
[269] Pea in (0, 3) moved to (0, 4).
[271] Pea in (0, 4) moved to (0, 5).
[273] Pea in (0, 5) moved to (0, 6).
[273] Pea in (0, 6) attacked.
[273] Zombie in (0, 6) was damaged. HP: 101.
[273] Pea in (0, 6) removed.
[278] Pea generated by Peashooter in (0, 0).
[278] Pea in (0, 0) moved to (0, 1).
[280] Pea in (0, 1) moved to (0, 2).
[282] Pea in (0, 2) moved to (0, 3).
[284] Pea in (0, 3) moved to (0, 4).
[286] Pea in (0, 4) moved to (0, 5).
[288] Pea in (0, 5) moved to (0, 6).
[288] Pea in (0, 6) attacked.
[288] Zombie in (0, 6) was damaged. HP: 81.
[288] Pea in (0, 6) removed.
[288] Zombie in (0, 6) moved to (0, 5).
[293] Pea generated by Peashooter in (0, 0).
[293] Pea in (0, 0) moved to (0, 1).
[294] Zombie in (0, 8) moved to (0, 7).
[295] Pea in (0, 1) moved to (0, 2).
[297] Pea in (0, 2) moved to (0, 3).
[299] Pea in (0, 3) moved to (0, 4).
[300] Zombie being spawned at lane 0
[301] Pea in (0, 4) moved to (0, 5).
[301] Pea in (0, 5) attacked.
[301] Zombie in (0, 5) was damaged. HP: 61.
[301] Pea in (0, 5) removed.
[303] Sun autospawned. Total: 25.
[308] Pea generated by Peashooter in (0, 0).
[308] Pea in (0, 0) moved to (0, 1).
[310] Pea in (0, 1) moved to (0, 2).
[312] Pea in (0, 2) moved to (0, 3).
[314] Pea in (0, 3) moved to (0, 4).
[316] Pea in (0, 4) moved to (0, 5).
[316] Pea in (0, 5) attacked.
[316] Zombie in (0, 5) was damaged. HP: 41.
[316] Pea in (0, 5) removed.
[323] Pea generated by Peashooter in (0, 0).
[323] Pea in (0, 0) moved to (0, 1).
[325] Pea in (0, 1) moved to (0, 2).
[327] Pea in (0, 2) moved to (0, 3).
[329] Pea in (0, 3) moved to (0, 4).
[331] Pea in (0, 4) moved to (0, 5).
[331] Pea in (0, 5) attacked.
[331] Zombie in (0, 5) was damaged. HP: 21.
[331] Pea in (0, 5) removed.
[335] Zombie in (0, 5) moved to (0, 4).
[338] Pea generated by Peashooter in (0, 0).
[338] Pea in (0, 0) moved to (0, 1).
[340] Pea in (0, 1) moved to (0, 2).
[341] Zombie in (0, 7) moved to (0, 6).
[342] Pea in (0, 2) moved to (0, 3).
[344] Pea in (0, 3) moved to (0, 4).
[344] Pea in (0, 4) attacked.
[344] Zombie in (0, 4) was damaged. HP: 1.
[344] Pea in (0, 4) removed.
[347] Zombie in (0, 9) moved to (0, 8).
[353] Pea generated by Peashooter in (0, 0).
[353] Pea in (0, 0) moved to (0, 1).
[355] Pea in (0, 1) moved to (0, 2).
[357] Pea in (0, 2) moved to (0, 3).
[359] Pea in (0, 3) moved to (0, 4).
[359] Pea in (0, 4) attacked.
[359] Zombie in (0, 4) was damaged. HP: -19.
[359] Zombie in (0, 4) was killed.
[359] Pea in (0, 4) removed.
[368] Pea generated by Peashooter in (0, 0).
[368] Pea in (0, 0) moved to (0, 1).
[370] Pea in (0, 1) moved to (0, 2).
[372] Pea in (0, 2) moved to (0, 3).
[374] Pea in (0, 3) moved to (0, 4).
[376] Pea in (0, 4) moved to (0, 5).
[378] Pea in (0, 5) moved to (0, 6).
[378] Pea in (0, 6) attacked.
[378] Zombie in (0, 6) was damaged. HP: 161.
[378] Pea in (0, 6) removed.
[383] Pea generated by Peashooter in (0, 0).
[383] Pea in (0, 0) moved to (0, 1).
[385] Pea in (0, 1) moved to (0, 2).
[387] Pea in (0, 2) moved to (0, 3).
[388] Zombie in (0, 6) moved to (0, 5).
[389] Pea in (0, 3) moved to (0, 4).
[391] Pea in (0, 4) moved to (0, 5).
[391] Pea in (0, 5) attacked.
[391] Zombie in (0, 5) was damaged. HP: 141.
[391] Pea in (0, 5) removed.
[394] Zombie in (0, 8) moved to (0, 7).
[398] Pea generated by Peashooter in (0, 0).
[398] Pea in (0, 0) moved to (0, 1).
[400] Pea in (0, 1) moved to (0, 2).
[400] Zombie being spawned at lane 0
[402] Pea in (0, 2) moved to (0, 3).
[404] Pea in (0, 3) moved to (0, 4).
[404] Sun autospawned. Total: 50.
[406] Pea in (0, 4) moved to (0, 5).
[406] Pea in (0, 5) attacked.
[406] Zombie in (0, 5) was damaged. HP: 121.
[406] Pea in (0, 5) removed.
[413] Pea generated by Peashooter in (0, 0).
[413] Pea in (0, 0) moved to (0, 1).
[415] Pea in (0, 1) moved to (0, 2).
[417] Pea in (0, 2) moved to (0, 3).
[419] Pea in (0, 3) moved to (0, 4).
[421] Pea in (0, 4) moved to (0, 5).
[421] Pea in (0, 5) attacked.
[421] Zombie in (0, 5) was damaged. HP: 101.
[421] Pea in (0, 5) removed.
[428] Pea generated by Peashooter in (0, 0).
[428] Pea in (0, 0) moved to (0, 1).
[430] Pea in (0, 1) moved to (0, 2).
[432] Pea in (0, 2) moved to (0, 3).
[434] Pea in (0, 3) moved to (0, 4).
[435] Zombie in (0, 5) moved to (0, 4).
[436] Pea in (0, 4) attacked.
[436] Zombie in (0, 4) was damaged. HP: 81.
[436] Pea in (0, 4) removed.
[441] Zombie in (0, 7) moved to (0, 6).
[443] Pea generated by Peashooter in (0, 0).
[443] Pea in (0, 0) moved to (0, 1).
[445] Pea in (0, 1) moved to (0, 2).
[447] Pea in (0, 2) moved to (0, 3).
[447] Zombie in (0, 9) moved to (0, 8).
[449] Pea in (0, 3) moved to (0, 4).
[449] Pea in (0, 4) attacked.
[449] Zombie in (0, 4) was damaged. HP: 61.
[449] Pea in (0, 4) removed.
[458] Pea generated by Peashooter in (0, 0).
[458] Pea in (0, 0) moved to (0, 1).
[460] Pea in (0, 1) moved to (0, 2).
[462] Pea in (0, 2) moved to (0, 3).
[464] Pea in (0, 3) moved to (0, 4).
[464] Pea in (0, 4) attacked.
[464] Zombie in (0, 4) was damaged. HP: 41.
[464] Pea in (0, 4) removed.
[473] Pea generated by Peashooter in (0, 0).
[473] Pea in (0, 0) moved to (0, 1).
[475] Pea in (0, 1) moved to (0, 2).
[477] Pea in (0, 2) moved to (0, 3).
[479] Pea in (0, 3) moved to (0, 4).
[479] Pea in (0, 4) attacked.
[479] Zombie in (0, 4) was damaged. HP: 21.
[479] Pea in (0, 4) removed.
[482] Zombie in (0, 4) moved to (0, 3).
[488] Pea generated by Peashooter in (0, 0).
[488] Pea in (0, 0) moved to (0, 1).
[488] Zombie in (0, 6) moved to (0, 5).
[490] Pea in (0, 1) moved to (0, 2).
[492] Pea in (0, 2) moved to (0, 3).
[492] Pea in (0, 3) attacked.
[492] Zombie in (0, 3) was damaged. HP: 1.
[492] Pea in (0, 3) removed.
[494] Zombie in (0, 8) moved to (0, 7).
[500] Zombie being spawned at lane 0
[503] Pea generated by Peashooter in (0, 0).
[503] Pea in (0, 0) moved to (0, 1).
[505] Pea in (0, 1) moved to (0, 2).
[505] Sun autospawned. Total: 75.
[507] Pea in (0, 2) moved to (0, 3).
[507] Pea in (0, 3) attacked.
[507] Zombie in (0, 3) was damaged. HP: -19.
[507] Zombie in (0, 3) was killed.
[507] Pea in (0, 3) removed.
[518] Pea generated by Peashooter in (0, 0).
[518] Pea in (0, 0) moved to (0, 1).
[520] Pea in (0, 1) moved to (0, 2).
[522] Pea in (0, 2) moved to (0, 3).
[524] Pea in (0, 3) moved to (0, 4).
[526] Pea in (0, 4) moved to (0, 5).
[526] Pea in (0, 5) attacked.
[526] Zombie in (0, 5) was damaged. HP: 161.
[526] Pea in (0, 5) removed.
[533] Pea generated by Peashooter in (0, 0).
[533] Pea in (0, 0) moved to (0, 1).
[535] Pea in (0, 1) moved to (0, 2).
[535] Zombie in (0, 5) moved to (0, 4).
[537] Pea in (0, 2) moved to (0, 3).
[539] Pea in (0, 3) moved to (0, 4).
[539] Pea in (0, 4) attacked.
[539] Zombie in (0, 4) was damaged. HP: 141.
[539] Pea in (0, 4) removed.
[541] Zombie in (0, 7) moved to (0, 6).
[547] Zombie in (0, 9) moved to (0, 8).
[548] Pea generated by Peashooter in (0, 0).
[548] Pea in (0, 0) moved to (0, 1).
[550] Pea in (0, 1) moved to (0, 2).
[552] Pea in (0, 2) moved to (0, 3).
[554] Pea in (0, 3) moved to (0, 4).
[554] Pea in (0, 4) attacked.
[554] Zombie in (0, 4) was damaged. HP: 121.
[554] Pea in (0, 4) removed.
[563] Pea generated by Peashooter in (0, 0).
[563] Pea in (0, 0) moved to (0, 1).
[565] Pea in (0, 1) moved to (0, 2).
[567] Pea in (0, 2) moved to (0, 3).
[569] Pea in (0, 3) moved to (0, 4).
[569] Pea in (0, 4) attacked.
[569] Zombie in (0, 4) was damaged. HP: 101.
[569] Pea in (0, 4) removed.
[578] Pea generated by Peashooter in (0, 0).
[578] Pea in (0, 0) moved to (0, 1).
[580] Pea in (0, 1) moved to (0, 2).
[582] Pea in (0, 2) moved to (0, 3).
[582] Zombie in (0, 4) moved to (0, 3).
[583] Pea in (0, 3) attacked.
[583] Zombie in (0, 3) was damaged. HP: 81.
[583] Pea in (0, 3) removed.
[588] Zombie in (0, 6) moved to (0, 5).
[593] Pea generated by Peashooter in (0, 0).
[593] Pea in (0, 0) moved to (0, 1).
[594] Zombie in (0, 8) moved to (0, 7).
[595] Pea in (0, 1) moved to (0, 2).
[597] Pea in (0, 2) moved to (0, 3).
[597] Pea in (0, 3) attacked.
[597] Zombie in (0, 3) was damaged. HP: 61.
[597] Pea in (0, 3) removed.
[600] Zombie being spawned at lane 0
[606] Sun autospawned. Total: 100.
[607] Planted Peashooter in (0, 1). Total: 0.
[608] Pea generated by Peashooter in (0, 0).
[608] Pea in (0, 0) moved to (0, 1).
[610] Pea in (0, 1) moved to (0, 2).
[612] Pea in (0, 2) moved to (0, 3).
[612] Pea in (0, 3) attacked.
[612] Zombie in (0, 3) was damaged. HP: 41.
[612] Pea in (0, 3) removed.
[622] Pea generated by Peashooter in (0, 1).
[622] Pea in (0, 1) moved to (0, 2).
[623] Pea generated by Peashooter in (0, 0).
[623] Pea in (0, 0) moved to (0, 1).
[624] Pea in (0, 2) moved to (0, 3).
[624] Pea in (0, 3) attacked.
[624] Zombie in (0, 3) was damaged. HP: 21.
[624] Pea in (0, 3) removed.
[625] Pea in (0, 1) moved to (0, 2).
[627] Pea in (0, 2) moved to (0, 3).
[627] Pea in (0, 3) attacked.
[627] Zombie in (0, 3) was damaged. HP: 1.
[627] Pea in (0, 3) removed.
[629] Zombie in (0, 3) moved to (0, 2).
[635] Zombie in (0, 5) moved to (0, 4).
[637] Pea generated by Peashooter in (0, 1).
[637] Pea in (0, 1) moved to (0, 2).
[637] Pea in (0, 2) attacked.
[637] Zombie in (0, 2) was damaged. HP: -19.
[637] Zombie in (0, 2) was killed.
[637] Pea in (0, 2) removed.
[638] Pea generated by Peashooter in (0, 0).
[638] Pea in (0, 0) moved to (0, 1).
[640] Pea in (0, 1) moved to (0, 2).
[641] Zombie in (0, 7) moved to (0, 6).
[642] Pea in (0, 2) moved to (0, 3).
[644] Pea in (0, 3) moved to (0, 4).
[644] Pea in (0, 4) attacked.
[644] Zombie in (0, 4) was damaged. HP: 161.
[644] Pea in (0, 4) removed.
[647] Zombie in (0, 9) moved to (0, 8).
[652] Pea generated by Peashooter in (0, 1).
[652] Pea in (0, 1) moved to (0, 2).
[653] Pea generated by Peashooter in (0, 0).
[653] Pea in (0, 0) moved to (0, 1).
[654] Pea in (0, 2) moved to (0, 3).
[655] Pea in (0, 1) moved to (0, 2).
[656] Pea in (0, 3) moved to (0, 4).
[656] Pea in (0, 4) attacked.
[656] Zombie in (0, 4) was damaged. HP: 141.
[656] Pea in (0, 4) removed.
[657] Pea in (0, 2) moved to (0, 3).
[659] Pea in (0, 3) moved to (0, 4).
[659] Pea in (0, 4) attacked.
[659] Zombie in (0, 4) was damaged. HP: 121.
[659] Pea in (0, 4) removed.
[667] Pea generated by Peashooter in (0, 1).
[667] Pea in (0, 1) moved to (0, 2).
[668] Pea generated by Peashooter in (0, 0).
[668] Pea in (0, 0) moved to (0, 1).
[669] Pea in (0, 2) moved to (0, 3).
[670] Pea in (0, 1) moved to (0, 2).
[671] Pea in (0, 3) moved to (0, 4).
[671] Pea in (0, 4) attacked.
[671] Zombie in (0, 4) was damaged. HP: 101.
[671] Pea in (0, 4) removed.
[672] Pea in (0, 2) moved to (0, 3).
[674] Pea in (0, 3) moved to (0, 4).
[674] Pea in (0, 4) attacked.
[674] Zombie in (0, 4) was damaged. HP: 81.
[674] Pea in (0, 4) removed.
[682] Pea generated by Peashooter in (0, 1).
[682] Pea in (0, 1) moved to (0, 2).
[682] Zombie in (0, 4) moved to (0, 3).
[683] Pea generated by Peashooter in (0, 0).
[683] Pea in (0, 0) moved to (0, 1).
[684] Pea in (0, 2) moved to (0, 3).
[684] Pea in (0, 3) attacked.
[684] Zombie in (0, 3) was damaged. HP: 61.
[684] Pea in (0, 3) removed.
[685] Pea in (0, 1) moved to (0, 2).
[687] Pea in (0, 2) moved to (0, 3).
[687] Pea in (0, 3) attacked.
[687] Zombie in (0, 3) was damaged. HP: 41.
[687] Pea in (0, 3) removed.
[688] Zombie in (0, 6) moved to (0, 5).
[694] Zombie in (0, 8) moved to (0, 7).
[697] Pea generated by Peashooter in (0, 1).
[697] Pea in (0, 1) moved to (0, 2).
[698] Pea generated by Peashooter in (0, 0).
[698] Pea in (0, 0) moved to (0, 1).
[699] Pea in (0, 2) moved to (0, 3).
[699] Pea in (0, 3) attacked.
[699] Zombie in (0, 3) was damaged. HP: 21.
[699] Pea in (0, 3) removed.
[700] Pea in (0, 1) moved to (0, 2).
[702] Pea in (0, 2) moved to (0, 3).
[702] Pea in (0, 3) attacked.
[702] Zombie in (0, 3) was damaged. HP: 1.
[702] Pea in (0, 3) removed.
[707] Sun autospawned. Total: 25.
[712] Pea generated by Peashooter in (0, 1).
[712] Pea in (0, 1) moved to (0, 2).
[713] Pea generated by Peashooter in (0, 0).
[713] Pea in (0, 0) moved to (0, 1).
[714] Pea in (0, 2) moved to (0, 3).
[714] Pea in (0, 3) attacked.
[714] Zombie in (0, 3) was damaged. HP: -19.
[714] Zombie in (0, 3) was killed.
[714] Pea in (0, 3) removed.
[715] Pea in (0, 1) moved to (0, 2).
[717] Pea in (0, 2) moved to (0, 3).
[719] Pea in (0, 3) moved to (0, 4).
[721] Pea in (0, 4) moved to (0, 5).
[721] Pea in (0, 5) attacked.
[721] Zombie in (0, 5) was damaged. HP: 161.
[721] Pea in (0, 5) removed.
[727] Pea generated by Peashooter in (0, 1).
[727] Pea in (0, 1) moved to (0, 2).
[728] Pea generated by Peashooter in (0, 0).
[728] Pea in (0, 0) moved to (0, 1).
[729] Pea in (0, 2) moved to (0, 3).
[730] Pea in (0, 1) moved to (0, 2).
[731] Pea in (0, 3) moved to (0, 4).
[732] Pea in (0, 2) moved to (0, 3).
[733] Pea in (0, 4) moved to (0, 5).
[733] Pea in (0, 5) attacked.
[733] Zombie in (0, 5) was damaged. HP: 141.
[733] Pea in (0, 5) removed.
[734] Pea in (0, 3) moved to (0, 4).
[735] Zombie in (0, 5) moved to (0, 4).
[736] Pea in (0, 4) attacked.
[736] Zombie in (0, 4) was damaged. HP: 121.
[736] Pea in (0, 4) removed.
[741] Zombie in (0, 7) moved to (0, 6).
[742] Pea generated by Peashooter in (0, 1).
[742] Pea in (0, 1) moved to (0, 2).
[743] Pea generated by Peashooter in (0, 0).
[743] Pea in (0, 0) moved to (0, 1).
[744] Pea in (0, 2) moved to (0, 3).
[745] Pea in (0, 1) moved to (0, 2).
[746] Pea in (0, 3) moved to (0, 4).
[746] Pea in (0, 4) attacked.
[746] Zombie in (0, 4) was damaged. HP: 101.
[746] Pea in (0, 4) removed.
[747] Pea in (0, 2) moved to (0, 3).
[749] Pea in (0, 3) moved to (0, 4).
[749] Pea in (0, 4) attacked.
[749] Zombie in (0, 4) was damaged. HP: 81.
[749] Pea in (0, 4) removed.
[757] Pea generated by Peashooter in (0, 1).
[757] Pea in (0, 1) moved to (0, 2).
[758] Pea generated by Peashooter in (0, 0).
[758] Pea in (0, 0) moved to (0, 1).
[759] Pea in (0, 2) moved to (0, 3).
[760] Pea in (0, 1) moved to (0, 2).
[761] Pea in (0, 3) moved to (0, 4).
[761] Pea in (0, 4) attacked.
[761] Zombie in (0, 4) was damaged. HP: 61.
[761] Pea in (0, 4) removed.
[762] Pea in (0, 2) moved to (0, 3).
[764] Pea in (0, 3) moved to (0, 4).
[764] Pea in (0, 4) attacked.
[764] Zombie in (0, 4) was damaged. HP: 41.
[764] Pea in (0, 4) removed.
[772] Pea generated by Peashooter in (0, 1).
[772] Pea in (0, 1) moved to (0, 2).
[773] Pea generated by Peashooter in (0, 0).
[773] Pea in (0, 0) moved to (0, 1).
[774] Pea in (0, 2) moved to (0, 3).
[775] Pea in (0, 1) moved to (0, 2).
[776] Pea in (0, 3) moved to (0, 4).
[776] Pea in (0, 4) attacked.
[776] Zombie in (0, 4) was damaged. HP: 21.
[776] Pea in (0, 4) removed.
[777] Pea in (0, 2) moved to (0, 3).
[779] Pea in (0, 3) moved to (0, 4).
[779] Pea in (0, 4) attacked.
[779] Zombie in (0, 4) was damaged. HP: 1.
[779] Pea in (0, 4) removed.
[782] Zombie in (0, 4) moved to (0, 3).
[787] Pea generated by Peashooter in (0, 1).
[787] Pea in (0, 1) moved to (0, 2).
[788] Pea generated by Peashooter in (0, 0).
[788] Pea in (0, 0) moved to (0, 1).
[788] Zombie in (0, 6) moved to (0, 5).
[789] Pea in (0, 2) moved to (0, 3).
[789] Pea in (0, 3) attacked.
[789] Zombie in (0, 3) was damaged. HP: -19.
[789] Zombie in (0, 3) was killed.
[789] Pea in (0, 3) removed.
[790] Pea in (0, 1) moved to (0, 2).
[792] Pea in (0, 2) moved to (0, 3).
[794] Pea in (0, 3) moved to (0, 4).
[796] Pea in (0, 4) moved to (0, 5).
[796] Pea in (0, 5) attacked.
[796] Zombie in (0, 5) was damaged. HP: 161.
[796] Pea in (0, 5) removed.
[802] Pea generated by Peashooter in (0, 1).
[802] Pea in (0, 1) moved to (0, 2).
[803] Pea generated by Peashooter in (0, 0).
[803] Pea in (0, 0) moved to (0, 1).
[804] Pea in (0, 2) moved to (0, 3).
[805] Pea in (0, 1) moved to (0, 2).
[806] Pea in (0, 3) moved to (0, 4).
[807] Pea in (0, 2) moved to (0, 3).
[808] Pea in (0, 4) moved to (0, 5).
[808] Pea in (0, 5) attacked.
[808] Zombie in (0, 5) was damaged. HP: 141.
[808] Pea in (0, 5) removed.
[808] Sun autospawned. Total: 50.
[809] Pea in (0, 3) moved to (0, 4).
[811] Pea in (0, 4) moved to (0, 5).
[811] Pea in (0, 5) attacked.
[811] Zombie in (0, 5) was damaged. HP: 121.
[811] Pea in (0, 5) removed.
[817] Pea generated by Peashooter in (0, 1).
[817] Pea in (0, 1) moved to (0, 2).
[818] Pea generated by Peashooter in (0, 0).
[818] Pea in (0, 0) moved to (0, 1).
[819] Pea in (0, 2) moved to (0, 3).
[820] Pea in (0, 1) moved to (0, 2).
[821] Pea in (0, 3) moved to (0, 4).
[822] Pea in (0, 2) moved to (0, 3).
[823] Pea in (0, 4) moved to (0, 5).
[823] Pea in (0, 5) attacked.
[823] Zombie in (0, 5) was damaged. HP: 101.
[823] Pea in (0, 5) removed.
[824] Pea in (0, 3) moved to (0, 4).
[826] Pea in (0, 4) moved to (0, 5).
[826] Pea in (0, 5) attacked.
[826] Zombie in (0, 5) was damaged. HP: 81.
[826] Pea in (0, 5) removed.
[832] Pea generated by Peashooter in (0, 1).
[832] Pea in (0, 1) moved to (0, 2).
[833] Pea generated by Peashooter in (0, 0).
[833] Pea in (0, 0) moved to (0, 1).
[834] Pea in (0, 2) moved to (0, 3).
[835] Pea in (0, 1) moved to (0, 2).
[835] Zombie in (0, 5) moved to (0, 4).
[836] Pea in (0, 3) moved to (0, 4).
[836] Pea in (0, 4) attacked.
[836] Zombie in (0, 4) was damaged. HP: 61.
[836] Pea in (0, 4) removed.
[837] Pea in (0, 2) moved to (0, 3).
[839] Pea in (0, 3) moved to (0, 4).
[839] Pea in (0, 4) attacked.
[839] Zombie in (0, 4) was damaged. HP: 41.
[839] Pea in (0, 4) removed.
[847] Pea generated by Peashooter in (0, 1).
[847] Pea in (0, 1) moved to (0, 2).
[848] Pea generated by Peashooter in (0, 0).
[848] Pea in (0, 0) moved to (0, 1).
[849] Pea in (0, 2) moved to (0, 3).
[850] Pea in (0, 1) moved to (0, 2).
[851] Pea in (0, 3) moved to (0, 4).
[851] Pea in (0, 4) attacked.
[851] Zombie in (0, 4) was damaged. HP: 21.
[851] Pea in (0, 4) removed.
[852] Pea in (0, 2) moved to (0, 3).
[854] Pea in (0, 3) moved to (0, 4).
[854] Pea in (0, 4) attacked.
[854] Zombie in (0, 4) was damaged. HP: 1.
[854] Pea in (0, 4) removed.
[862] Pea generated by Peashooter in (0, 1).
[862] Pea in (0, 1) moved to (0, 2).
[863] Pea generated by Peashooter in (0, 0).
[863] Pea in (0, 0) moved to (0, 1).
[864] Pea in (0, 2) moved to (0, 3).
[865] Pea in (0, 1) moved to (0, 2).
[866] Pea in (0, 3) moved to (0, 4).
[866] Pea in (0, 4) attacked.
[866] Zombie in (0, 4) was damaged. HP: -19.
[866] Zombie in (0, 4) was killed.
[866] Pea in (0, 4) removed.
[866] You've won!
